tests/acceptance/test_project_ownership.py

```diff
@@ -19,4 +19,5 @@ def test_open_modal(self):
         self.browser.wait_until_test_id("issueowners-panel")
         self.browser.click('[aria-label="Edit"]')
         self.browser.wait_until(".modal-dialog")
+        self.browser.wait_until_not("div[class$='loadingIndicator']")
```

Comment:
Is this because it's an emotion class name?

Reply:
@billyvg yes. this matches on a partial classname.